src/Containers/App/index.jsx

```diff
@@ -777,7 +777,36 @@ class App extends Component {
     }
 
     try {
-      const apiVersion = await this.serial.getApiVersion();
+      const apiVersion = await this.serial.getApiVersion().catch(async (e) => {
+        if (e.message.includes("Timed out after")) {
+          let hasResets = false;
+          let i = 0;
+          let esc = await this.serial.getFourWayInterfaceInfo(i).catch(() => null);
+          while (esc !== null) {
+
+            if (!esc) {
+              break;
+            }
+
+            try {
+              await this.serial.resetFourWayInterface(i);
+              hasResets = true;
+            } catch(e) {
+              this.addLogMessage('resetEscFailed', { index: i + 1 });
```

Comment:
Does it even make sense to continue if we failed resetting one ESC? I think it would be then better to tell the user to power cycle.

Reply:
changed to that